tests/foreman/cli/test_domain.py

```diff
@@ -38,6 +39,50 @@ def test_positive_create(self, options):
         except CLIFactoryError as err:
             self.fail(err)
 
+    def test_create_domain_with_location(self):
+        """@Test: Check if domain with location can be created
+
+        @Feature: Domain - Positive create
+
+        @Assert: Domain is created and has new location assigned
+
+        """
+        try:
+            location = make_location()
+            domain = make_domain({
+                'location-ids': location['id']
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        self.assertIn(
+            location['name'],
+            domain['locations'],
+            "Location wasn't assigned to domain"
```

Comment:
@elyezer, do you have an opinion on whether to keep the two error messages in this PR? I prefer to omit them, but you've been spending more time with the CLI test suite than me, so your opinion is more relevant.